lib/logstash/filters/jdbc_static.rb

```diff
@@ -214,6 +218,7 @@ def global_lookup_options(options = Hash.new)
     options["lookup_jdbc_driver_class"] = @lookup_jdbc_driver_class
     options["lookup_jdbc_driver_library"] = @lookup_jdbc_driver_library
     options["lookup_jdbc_connection_string"] = @lookup_jdbc_connection_string
+    options["ecs_compatibility"] = @ecs_compatibility
```

Comment:
believe [we should not rely](https://github.com/logstash-plugins/logstash-mixin-ecs_compatibility_support/blob/v1.2.0/lib/logstash/plugin_mixins/ecs_compatibility_support.rb#L69-L71) on the raw value but instead use the accessor method : 
```suggestion
    options["ecs_compatibility"] = ecs_compatibility
```
... while this works with the LegacyAdapter LS' implementation does not have the same value for `@ecs_compatibility` and the (accessor) `ecs_compatibility` method